spec/javascripts/app/views/help_view_spec.js

```diff
@@ -134,4 +141,47 @@ describe("app.views.Help", function(){
       });
     });
   });
+
+  describe("chat section", function(){
+    describe("chat enabled", function(){
+      beforeEach(function(){
+        gon.chatEnabled = true;
+<<<<<<< HEAD
```

Comment:
Expected '<<' to have an indentation at 9 instead at 1.<br>Expected an identifier and instead saw '<<'.<br>Expected an operator and instead saw '<<'.<br>Expected an operator and instead saw '<'.<br>Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.